public/class-lnd-for-wp-public.php

```diff
@@ -91,23 +90,23 @@ public function enqueue_scripts() {
 	 */
 	public function request_lightning_invoice_ajax() {
 
-		if(
-			isset( $_REQUEST['lnd-post-nonce'] ) &&
-			wp_verify_nonce( $_REQUEST['lnd-post-nonce'] , 'lnd_request_invoice' )
-		){
+		//if(
+		//	isset( $_REQUEST['lnd-post-nonce'] ) &&
+		//	wp_verify_nonce( $_REQUEST['lnd-post-nonce'] , 'lnd_request_invoice' )
+		//){
```

Comment:
We need to include wp_verify_nonce on all form request handlers. Can you un-comment these?

Reply:
In a later commit this lines are un-comment.